jobs/kpi-forecasting/kpi-forecasting/Utils/PosteriorSampling.py

```diff
@@ -71,6 +70,8 @@ def get_confidence_intervals(
         columns={"y": "value"}
     ).sort_values(by="{}".format(aggregation_unit_of_time))
 
+    observed_aggregated = observed_aggregated.astype({"value": np.float64})
```

Comment:
`observed_aggregated["value"]` is being stored as an `Int64Dtype`, which is a pandas type for storing large integers. For some reason, using this type breaks the following merge on line 100:
```python
   all_aggregated = pd.merge(
        observed_aggregated,
        uncertainty_samples_aggregated,
        on=["{}".format(aggregation_unit_of_time), "value", "type"],
        how="outer",
    )
```
I think using float64 instead is an okay workaround here, since the values in the confidence intervals are reported as float64 anyways.